src/Layout/Presentation/Presentation.tsx

```diff
@@ -55,8 +55,8 @@ export class Presentation extends React.Component<PresentationProps, {}> {
     getSeatDistributionData(): DistrictResult[] {
         return getSeatDistributionData(
             this.props.results.districtResults,
+            this.props.showPartiesWithoutSeats,
             this.props.results.partyResults,
-            this.props.showPartiesWithoutSeats
```

Comment:
This is a good example of why one should try to avoid functions with more than two inputs. 😁

I would have to look up the documentation for this function to know what the different things meant and what the ordering was (and why this fixed it).